apps/site/src/components/pages/wordpress/voting/applications.tsx

```diff
@@ -0,0 +1,63 @@
+import { ReactElement } from "react";
+import {
+  CommentDotsIcon,
+  ContentfulIcon,
+  GithubIcon,
+  SanityIcon,
+  StrapiIcon,
+} from "../../../icons";
+
+export enum ApplicationIds {
```

Comment:
It'd be great to avoid enums if possible. We can use discriminated unions instead. [Internal Slack thread](https://hashintel.slack.com/archives/C02TWBTT3ED/p1667303607693499).

Reply:
Here, you can use  

```
export interface ApplicationInfo {}

const applicationInfoById = {
 "ECO_SANITY": {...}
} satisfies Record<string, ApplicationInfo>

Object.entries(applicationInfoById).forEach([applicationId, applicationInfo])
```

Reply:
I see some value in this. We are likely to use the voting component again in the future, in other contexts.

Reply:
Addressed in eef141d6028484a0c3393645292c18a6bced3fc3. Thank you